src/fmu/dataio/datastructure/meta/meta.py

```diff
@@ -186,71 +201,113 @@ class RealizationJobListing(BaseModel):
 
 
 class Realization(BaseModel):
-    id: int = Field(
-        description="The unique number of this realization as used in FMU",
-    )
-    name: str = Field(
-        description="The convential name of this iteration, e.g. iter-0 or pred",
-        examples=["iter-0"],
-    )
-    parameters: Optional[Parameters] = Field(
-        default=None,
-        description="Parameters for this realization",
-    )
-    jobs: Optional[object] = Field(
-        default=None,
-        description=(
-            "Content directly taken from the ERT jobs.json file for this realization"
-        ),
-    )
+    """The ``fmu.realization`` block contains information about the realization this
+    data object belongs to."""
+
+    id: int
+    """The internal ID of the realization, typically represented by an integer."""
+
+    name: str = Field(examples=["iter-0"])
+    """The name of the realization. This is typically reflecting the folder name on
+    scratch. Custom names for realizations are not supported by ERT, but we still
+    recommend to use ``fmu.realization.id`` for all usage except purely visual
+    appearance."""
+
+    parameters: Optional[Parameters] = Field(default=None)
+    """These are the parameters used in this realization. It is a direct pass of
+    ``parameters.txt`` and will contain key:value pairs representing the design
+    parameters. See :class:`Parameters`."""
+
+    jobs: Optional[object] = Field(default=None)
+    """Content directly taken from the ERT jobs.json file for this realization."""
+
     uuid: UUID = Field(examples=["15ce3b84-766f-4c93-9050-b154861f9100"])
+    """The universally unique identifier for this realization. It is a hash of
+    ``fmu.case.uuid`` and ``fmu.iteration.uuid`` and ``fmu.realization.id``."""
 
 
 class CountryItem(BaseModel):
-    identifier: str = Field(
-        examples=["Norway"],
-    )
+    """Referrence to a country known to SMDA."""
+
+    identifier: str = Field(examples=["Norway"])
+    """Identifier known to SMDA."""
+
     uuid: UUID = Field(examples=["15ce3b84-766f-4c93-9050-b154861f9100"])
+    """Identifier known to SMDA."""
 
 
 class DiscoveryItem(BaseModel):
-    short_identifier: str = Field(
-        examples=["SomeDiscovery"],
-    )
+    """Referrence to a discovery known to SMDA."""
```

Comment:
Typo?

Reply:
I see I copied and pasted this all over... fixed, thanks 👍